src/helps/cloudinaryConfig.ts

```diff
@@ -0,0 +1,12 @@
+import { v2 as cloudinary } from 'cloudinary';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+cloudinary.config({
+  cloud_name: process.env.CLOUDINARY_NAME,
+  api_key: process.env.CLOUDINARY_API_KEY,
+  api_secret: process.env.CLOUDINARY_API_SECRET,
```

Comment:
Update .env.example with these! Add any other environment variable that you are using too

Reply:
Updated